javascript/node/events/listening-to-events-just-once.md

```diff
@@ -34,32 +28,39 @@ aspects:
 ---
 ## Content
 
-If more than one uncaught exception was to be caught, the applications equivalent to a `closedown` method, where the application is shutdown, would be triggered more than once, consequently causing issues with the applications shutdown procedure.
+Some applications can include a special behavior that happens whenever it is closed. This behavior can be used to free up resources, close a connection to a databases (or anything else), and for the remainder of this insight we will call it `closedown`.
 
-To only call the `closedown`  method once, only the first instance of an uncaught exception being caught should be used. This is done using `emitter.once` , for example, if an application's emitter was to be called *test*:
-```javascript
-test.once(‘uncaughtException’,
-                  function(err) {
+Suppose the application throws an unexpected exception, which is then caught followed by the `closedown` method being called before terminating the application (which is the expected behavior).
```

Comment:
We can remove this paragraph if [this comment](https://github.com/enkidevs/curriculum/pull/1778/files#r366351228) is applied